### DISCUSSION TITLE
password prompt like dolphin with polkit when using yazi?

### DISCUSSION THREAD

**Discussion by ingenarel** [sxyazi/yazi#1251] [4. Ideas]
so i need some help. i'm using yazi in arch and i edit protected files a lot. and i forget to include sudo every time i do that so i have to close yazi and then navigate through those files again. is there any way yazi can pop out password prompts like that?

here's an example when i try to mount something in dolphin:
![image](https://github.com/sxyazi/yazi/assets/145784829/de2280d6-deb6-4ad0-9784-b38eaa4390f6)

so is there any way to do that? it would be really great!

**Comment by sxyazi** [comment]
Have you tried the `sudo.yazi` plugin? https://github.com/TD-Sky/sudo.yazi
